### PR TITLE
zh:sync path

### PR DESCRIPTION
**Pull request by jzhupup**
Update [自动注入 Sidecar] path sync with the en version

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by zirain** [comment]
@jzhupup please join the community to skip the `ok-to-test` wait

File: content/zh/docs/examples/microservices-istio/add-istio/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ test: no
 1. 重新部署 `productpage` 微服务，启用 Istio：
 
     {{< tip >}}
-    本教程为了教学目的将会逐步演示手动注入 Sidecar 去启用 Istio，但是 [自动注入 Sidecar](/zh/docs/ops/configuration/mesh/injection-concepts/) 更加方便.
+    本教程为了教学目的将会逐步演示手动注入 Sidecar 去启用 Istio，但是 [自动注入 Sidecar](/zh/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection) 更加方便.
```
Comment (windsonsea) [inline content/zh/docs/examples/microservices-istio/add-istio/index.md:23]:
```suggestion
    本教程为了教学目的将会逐步演示如何手动注入 Sidecar 启用 Istio，但是[自动注入 Sidecar](/zh/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection) 更加便捷。
```

File: content/zh/docs/examples/microservices-istio/add-istio/index.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ test: no
         caption="在左上角的下拉菜单中，选择 Istio Mesh Dashboard"
         >}}
 
-    注意 在命名空间中的 `productpage` 服务，它的命名应该是 `productpage.<your namespace>.svc.cluster.local`。
+    注意在命名空间中的 `productpage` 服务，它的命名应该是 `productpage.<your namespace>.svc.cluster.local`。
```
Comment (windsonsea) [inline content/zh/docs/examples/microservices-istio/add-istio/index.md:84]:
```suggestion
    注意命名空间中的 `productpage` 服务，它的命名应该是 `productpage.<your namespace>.svc.cluster.local`。
```

**Review by windsonsea — CHANGES_REQUESTED** [review]

File: content/zh/docs/examples/microservices-istio/add-istio/index.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ test: no
         caption="在左上角的下拉菜单中，选择 Istio Mesh Dashboard"
         >}}
 
-    注意 在命名空间中的 `productpage` 服务，它的命名应该是 `productpage.<your namespace>.svc.cluster.local`。
+    注意在命名空间中 `productpage` 服务，它的命名应该是 `productpage.<your namespace>.svc.cluster.local`。
```
Comment (windsonsea) [inline content/zh/docs/examples/microservices-istio/add-istio/index.md:84]: If you partially accepted and modified with a comment, you cannot mark it as `resolved`. A proper action is to show your feedback, thanks.

**Review by windsonsea — CHANGES_REQUESTED** [review]

File: content/zh/docs/examples/microservices-istio/add-istio/index.md (hunk 3 above)
Comment (jzhupup) [reply content/zh/docs/examples/microservices-istio/add-istio/index.md]: I think the format is wrong, it has now been corrected.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**